README.md

```diff
@@ -1203,6 +1203,18 @@ netfilter hooks to Lua.
 * `"LOCAL_OUT"`: `NF_INET_LOCAL_OUT`. The packet is generated by the local system.
 * `"POST_ROUTING"`: `NF_INET_POST_ROUTING`. The packet is about to be sent out.
 
+### xtable userspace library (libxtable)
+
+The `libxtable` [userspace library](usr/lib/xtable) provides support for generating userspace code for [xtable extensions](https://inai.de/projects/xtables-addons/). The user can modify the generated lua code to implement the userspace handlers for the corresponding xtable extension.
+
+To generate the library, the following steps are required:
```

Comment:
I would document the API as well.. in the same way you have done for the kernel library

Reply:
Ok